macros/src/pro/mod.rs

```diff
@@ -0,0 +1,229 @@
+use crate::testing::{literal_to_fn, parse_config_args, ConfigArgs, Result, TestConfig};
+use proc_macro2::TokenStream;
+use quote::quote;
+use syn::ItemFn;
+
```

Comment:
there are still a few places where the macro is not used: quota_check, db_type, sentinel cogs

Jut ctrl+f "with_pro_temp_context"

Reply:
[a5f5151](https://github.com/geo-engine/geoengine/pull/859/commits/a5f51519fcf8ed23b9a16f43b92715ef8f81e190)

quota_check is a benchmark, I cannot use the test macro there